src/modules/homepage.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-plusplus */
```

Comment:
please it would be better to remove the diable linter cause it ensures that you write a clean code